hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -837,6 +838,7 @@ protected OmKeyInfo wrapUncommittedBlocksAsPseudoKey(
     LOG.info("Detect allocated but uncommitted blocks {} in key {}.",
         uncommitted, omKeyInfo.getKeyName());
     OmKeyInfo pseudoKeyInfo = omKeyInfo.copyObject();
+    pseudoKeyInfo.setObjectID(OBJECT_ID_DEFAULT);
```

Comment:
Let's put a comment above this line that we are using this as a special marker to indicate that this key's blocks can be unconditionally deleted during snapshot reclamation.